src/components/Player/Player.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/prop-types */
+import styled from '@emotion/styled';
+import YouTube from '@u-wave/react-youtube';
+import { useState } from 'react';
+
+type Props = {
+  videoId: string;
+  handleNext: () => void;
+  handleBefore: () => void;
+};
+export const Player: React.FC<Props> = ({
```

Comment:
```suggestion
export const YoutubePlayer: React.FC<Props> = ({
```

youtube만 재생 가능한 Player이니, 이름을 좀 더 specific하게 써주는 게 좋겠습니다